assets/js/components/ParticipantsList.tsx

```diff
@@ -24,31 +34,56 @@ const ModeratorMenu = ({ isPresenter }) => {
         backgroundColor="red"
       />
       <MenuList>
-        <MenuItem>{isPresenter ? "Set as a normal participant" : "Set as a presenter"}</MenuItem>
-        <MenuItem>Mute</MenuItem>
-        <MenuItem>Kick</MenuItem>
+        <MenuItem
+          onClick={handleClick}
+          value={isPresenter ? "Set as a normal participant" : "Set as a presenter"}
+        >
+          {isPresenter ? "Set as a normal participant" : "Set as a presenter"}
+        </MenuItem>
+        <MenuItem onClick={handleClick} value="Mute">
```

Comment:
Why do you use the same `onclick` function for all menuItems?

Reply:
Good point.